ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/isis_area_id.schema.yml

```
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  isis_area_id:
+    documentation_options:
+      filename: Fabric Variables
+      table: ISIS Settings
+    description: Required when "underlay_routing_protocol" == ISIS variants
+    type: str
```

This could be intepreted as a float or int, so I suggest we also convert from those.
```suggestion
    type: str
    convert_types:
      - int
      - float
```